python/xorbits/_mars/services/scheduling/worker/execution.py

```diff
@@ -478,6 +478,12 @@ async def _run_subtask_once():
                     # may encounter subprocess memory error
                     sub_pool_address = await slot_manager_ref.get_slot_address(slot_id)
                     await mo.wait_actor_pool_recovered(sub_pool_address, self.address)
+
+                if isinstance(ex, ServerClosed):
+                    # view the exception as a potential OOM problem temporarily
+                    ex = ServerClosed(
+                        f"unexpectedly terminated process ({ex}) and highly suspected to be caused by an Out-of-Memory (OOM) problem"
+                    )
```

Comment:
The exception information should preferably include node ip and process pid.

Reply:
#621 follows up.